### PR TITLE
Better meter titles

### PR DESCRIPTION
**Pull request by bitamar**
#726

### CODE REVIEW

**Comment by bitamar** [comment]
Ready for review

File: negawatt/modules/custom/negawatt_meter/negawatt_meter.drush.inc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+/**
+ * @file
+ * Drush integration for the Negawatt meter module.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function negawatt_meter_drush_command() {
+  $items['update-meter-titles'] = array(
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_meter/negawatt_meter.drush.inc]: should this be just a drush script? (instead of a command)

Comment (bitamar) [inline negawatt/modules/custom/negawatt_meter/negawatt_meter.drush.inc]: I'm not sure. Is there any downside to having it as a command?

File: negawatt/modules/custom/negawatt_meter/negawatt_meter.module [hunk 2]
```diff
@@ -267,6 +267,11 @@ function negawatt_meter_node_presave($node) {
   // Set meter category by trying to match description with categories'
   // match-strings.
   negawatt_meter_set_meter_category($node);
+
+  // Set default title.
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_meter/negawatt_meter.module]: explain.

File: negawatt/modules/custom/negawatt_meter/negawatt_meter.module [hunk 3]
```diff
@@ -445,3 +450,43 @@ function negawatt_meter_vocab_id_from_group_id($group_id) {
     ->fetchField();
   return $vocabulary_id;
 }
+
+/**
+ * Determine meter title according to its fields: Meter description, otherwise
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_meter/negawatt_meter.module]: one liner.

File: negawatt/modules/custom/negawatt_meter/negawatt_meter.module [hunk 4]
```diff
@@ -445,3 +450,43 @@ function negawatt_meter_vocab_id_from_group_id($group_id) {
...
+ *   A meter node.
+ *
+ * @return string
+ *   The meter title.
+ */
+function negawatt_meter_get_meter_title($node) {
+  $wrapper = entity_metadata_wrapper('node', $node);
+
+  if ($wrapper->field_place_description->value()) {
+    return $wrapper->field_place_description->value();
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_meter/negawatt_meter.module]: comment

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**